src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -181,11 +181,15 @@ const state = reactive({
     isFull: computed<boolean>(() => props.size === WIDGET_SIZE.full),
     dateLabel: computed<TranslateResult|undefined>(() => {
         const start = setBasicDateFormat(props.dateRange?.start);
-        const end = setBasicDateFormat(props.dateRange?.end);
-        if (start && end) {
-            return `${start} ~ ${end}`;
+        const endDayjs = props.dateRange?.end ? dayjs.utc(props.dateRange.end) : undefined;
+        if (start && endDayjs) {
+            let endText;
+            const isCurrentMonth = endDayjs.isSame(dayjs.utc(), 'month');
+            if (isCurrentMonth) endText = dayjs.utc().format('YYYY-MM-DD');
+            else endText = endDayjs.endOf('month').format('YYYY-MM-DD');
+            return `${start} ~ ${endText}`;
         }
```

Comment:
description:
if end is current, it returns today (ex.`2023-01-09`).
if not, it returns last day of end month (ed. `2022-12-31`).